### DISCUSSION TITLE
Welcome to liferea Discussions!

### DISCUSSION THREAD

**Discussion by lwindolf** [lwindolf/liferea#969] [General]
## Welcome!

Let's try to use discussions for 

  * Feature Requests
  * Concepts for new stuff
  * To share any ideas on how to use Liferea

Please open an [issue](https://github.com/lwindolf/liferea/issues) instead if you

  * Want to report a bug
  * Need help with Liferea features

**Comment by sunwire** [comment]
It would be a good idea to inform everyone at least one week before the release of a new version  so that we could update the translation if needed.

**Reply by lwindolf** [reply]
I 100% agree and I do practice this for release candidates (e.g. when the unstable line transistions to stable), but given my time budget I have to do the release when I have time and I simply cannot predict it. E.g. before this release there were 4 weeks of inactivity :-)

So someone else would have to take over release work for this to happen.

**Comment by Martin-Vermeer** [comment]
When an item in a feed has no title, it prints *** No title ***. Someone pointed out to me that this looks like an error message, but it seems to be 100% legal. E.g., Mastodon RSS feeds have untitled posts. Something more neutral, like (No title) ?

**Reply by Martin-Vermeer** [reply]
This post proposes an interesting alternative: use the first words of the content as title! Informative.
https://mastodon.social/@davew/109456451234589189

**Reply by lwindolf** [reply]
Using the first words of the text makes total sense.

**Reply by lwindolf** [reply]
Created issue https://github.com/lwindolf/liferea/issues/1332 to address this.
